src/parse-stream.js

```diff
@@ -363,6 +363,21 @@ export default class ParseStream extends Stream {
       this.trigger('data', event);
       return;
     }
+    match = (/^#EXT-X-START:?(.*)$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'start'
+      };
+      if (match[1]) {
+        event.attributes = parseAttributes(match[1]);
+
+        event.attributes['TIME-OFFSET'] = parseFloat(event.attributes['TIME-OFFSET'], 10);
```

Comment:
Does parseFloat take a second parameter?

Reply:
I don't believe it does - sorry about that, I'm used to `parseInt` taking a base parameter and just assumed `parseFloat` behaved similarly...